src/components/client/core/Drawer/recipe.ts

```diff
@@ -0,0 +1,95 @@
+import { sva } from "generated/panda/css";
```

Comment:
I think we should adopt the pattern of `[ComponentName].recipe.ts` (e.g. `Drawer.recipe.ts`) for recipe file names.

It looks like Panda CSS does this as well (example: https://panda-css.com/docs/concepts/slot-recipes#defining-the-recipe), though my preference would be to capitalize the name to embody the "React component" intent.